src/components/Col.js

```diff
@@ -26,18 +27,18 @@ module.exports = React.createClass({
 	},
 	getInitialState: function() {
 		return {
-			windowWidth: window.innerWidth
+			windowWidth: root.innerWidth
```

Comment:
Seems reasonable.  Although, using the other package (or something similar internal to elemental) allows for event listeners, etc. on the 'root' server-side.